awscli/customizations/paginate.py

```diff
@@ -337,8 +338,15 @@ def add_to_parser(self, parser):
         parser.add_argument(self.cli_name, dest=self.py_name,
                             type=self.type_map[self._parse_type])
 
+    def _emit_non_positive_max_items_warning(self):
+            uni_print(
+                "warning: Non-positive values for --max-items may result in undefined behavior.\n",
+                sys.stderr)
```

Comment:
nit: The formatting looks off here

Reply:
Updated in latest revision. Please let me know if we have a formatter / formatting guide anywhere for this repository.  I kept it split into 2 lines to avoid too-long of a line but not sure if we have formal requirements on that anywhere.